src/plugins/legacy/medFilteringWorkspaceL/medFilteringWorkspaceL.cpp

```diff
@@ -35,8 +35,6 @@ class medFilteringWorkspaceLPrivate
 medFilteringWorkspaceL::medFilteringWorkspaceL(QWidget *parent)
     : medSelectorWorkspace (parent, staticName()), d(new medFilteringWorkspaceLPrivate)
 {
-    connect(this->tabbedViewContainers(), SIGNAL(containersSelectedChanged()),
```

Comment:
I don't understand why it need to be removed

Reply:
inputChanged is already called in medFilteringWorkspaceL::changeToolBoxInput() below

Reply:
And this line emits inputChanged even if the output image is changed. We do not want that. That's why in medFilteringWorkspaceL::setupTabbedViewContainer() we choose what we want to do with either the input or the output container

Reply:
ok

Reply:
Indeed :)